config/common.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use League\Flysystem\Local\LocalFilesystemAdapter;
+use League\Flysystem\UnixVisibility\PortableVisibilityConverter;
+use Yiisoft\Files\FilesystemInterface;
+use Yiisoft\Files\Filesystem;
+use Yiisoft\Files\FileStorageConfigs;
+
+return [
+    FilesystemInterface::class => function () use ($params) {
+        $aliases = $params['aliases'] ?? [];
+        if (!isset($aliases['@root'])) {
+            throw new \InvalidArgumentException('Alias of the root directory is not defined.');
```

Comment:
this is more like a runtime exception or other possible config exception

Reply:
I agree